tools/cli/persistenceUtil.go

```diff
@@ -94,13 +96,12 @@ func CreateDefaultDBConfig(c *cli.Context) (config.DataStore, error) {
 			Keyspace: c.String(FlagKeyspace),
 			TLS:      tls,
 		}
-
 		defaultStore := config.DataStore{
 			Cassandra: defaultConfig,
 		}
-
 		return defaultStore, nil
-	} else if engine == mySQLDBType {
+	case mysql.PluginName:
+	case postgresql.PluginName:
```

Comment:
switch in go does not do fall through by default

try 

case mysql.PluginName, postgresql.PluginName:

Reply:
yes, addressed while testing. Thx